README.md

```diff
@@ -31,7 +31,7 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 * doc blocks of class properties must be multiline and have exactly one `@var` annotation
 
 ### MO4.Formatting.AlphabeticalUseStatements
-* `use` statements must be sorted lexicographically, grouped by empty lines. The order function can be configured.
```

Comment:
Grouping `use` statements is a crucial feature! It would be very sad if this feature gets removed 🙀

Reply:
And I would be very happy. So let's vote here:
:+1: merge as-is
:-1: remove this rule (also removing the ensured spacing of one newline before and after the whole uses block.

Reply:
Kind of sad that this doesn't get real support, so I'm backing it out.